frontend/yarn.lock

```diff
@@ -16,21 +16,21 @@
   dependencies:
     "@babel/highlight" "^7.10.4"
 
```

Comment:
do we know what we do here? or did you push the changed yarn.lock by accident?

Reply:
whoopsie, removing.